src/rewriters/Substitutor.h

```diff
@@ -46,4 +51,13 @@ class Substitutor : public Rewriter<SubstitutionConfig> {
     Substitutor(Logic &logic, SubstitutionConfig::SubMap const &substs) :
             Rewriter<SubstitutionConfig>(logic, config),
             config(logic, substs) {}
+};
+
+class IteSubstitutor : public Rewriter<IteSubstitutionConfig> {
```

Comment:
As mentioned before, I don't think we need this anymore.

Reply:
Postponed to the next PR.